lms/static/js/student_profile/views/learner_profile_fields.js

```diff
@@ -0,0 +1,38 @@
+;(function (define, undefined) {
+    'use strict';
+    define([
+        'gettext', 'jquery', 'underscore', 'backbone', 'js/views/fields', 'js/vendor/backbone-super'
+    ], function (gettext, $, _, Backbone, FieldViews) {
+
+        var LearnerProfileFieldViews = {};
+
+        LearnerProfileFieldViews.AccountPrivacyFieldView = FieldViews.DropdownFieldView.extend({
+
+            render: function () {
+                this._super();
+                this.message();
+                return this;
+            },
+
+            message: function () {
+                if (this.profileIsPrivate) {
+                    this._super(interpolate_text(
+                        gettext("Your profile is disabled because you haven't filled in your Year of Birth. Please visit the Account Settings page. {account_settings_page_link}."),
```

Comment:
You must specify your birth year before you can share your full profile. To specify your birth year, go to the Account Settings page. (make "Account Settings page" a link)